{
	"schema_version": "1.6.0",
	"id": "RHBA-2018:2796",
	"modified": "2024-09-13T16:48:05Z",
	"published": "2024-09-13T16:48:05Z",
	"related": [
		"CVE-2018-10892"
	],
	"summary": "Red Hat Bug Fix Advisory: podman bug fix update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:C/C:L/I:L/A:L"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:rhel_extras_other:7",
				"name": "podman",
				"purl": "pkg:rpm/redhat/podman"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:0.9.2-5.git37a2afe.el7_5"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHBA-2018:2796.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_extras_other:7",
				"name": "podman-debuginfo",
				"purl": "pkg:rpm/redhat/podman-debuginfo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:0.9.2-5.git37a2afe.el7_5"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHBA-2018:2796.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHBA-2018:2796"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1623985"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2018/rhba-2018_2796.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2018-10892"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1598581"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2018-10892"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2018-10892"
		}
	]
}
